device-types/Cisco/AIR-AP1562I-Q-K9.yaml

```diff
@@ -0,0 +1,17 @@
+---
+manufacturer: Cisco
+model: Aironet 1562I-Q-K9
+slug: cisco-air-ap1562i-q-k9
+part_number: AIR-AP1562I-Q-K9
+u_height: 0
+is_full_depth: false
```

Comment:
Weight is available, please include weight

Reply:
Will be added on next commit